universum/analyzers/uncrustify.py

```diff
@@ -1,6 +1,7 @@
 import argparse
 import difflib
 import sys
+from shutil import which
```

Comment:
I think `import shutil` & `shutil.which()` will be easier to read (as it will be more obvious that 'which' is not a locally declared function by reading the line).
The same goes with `os.environ`.
This approach is generally used in most Universum modules